src/IRMFactory/EulerKinkIRMFactory.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+pragma solidity ^0.8.0;
+
+import {IRMLinearKink} from "evk/InterestRateModels/IRMLinearKink.sol";
+
+/// @title EulerKinkIRMFactory
+/// @author Euler Labs (https://www.eulerlabs.com/)
+/// @notice A minimal factory for Kink IRMs.
+contract EulerKinkIRMFactory {
+    // corresponds to 1000% APY
+    uint256 internal constant MAX_ALLOWED_INTEREST_RATE = 75986276241127470105;
```

Comment:
Not that it matters in real life, but we should do this for full precision for consistency with the value from `Constants.sol`:

```
-> floor(((1000 / 100 + 1)**(1/(86400*365.2425)) - 1) * 1e27)
 = 75986279153383989049

```
Reference: https://www.wolframalpha.com/input?i=floor%28%28%281000+%2F+100+%2B+1%29**%281%2F%2886400*365.2425%29%29+-+1%29+*+1e27%29